cli/src/install.sh

```diff
@@ -53,12 +57,15 @@ add_to_path_and_alias() {
     shell=${2}
 
     # shellcheck disable=SC2016 # we don't want to expand $PATH here
-    if ! grep -q '.phylum:\$PATH' "${rc_path}"; then
```

Comment:
This script is also responsible for upgrade. And as a part of that upgrade, we need to remove the `export PATH="$HOME/.phylum:$PATH"` from the rc files that the previous `install.sh` may have put there.

The same goes for the lines added for tab completion below. We need to remove the old lines, if they exist.